### PR TITLE
Update `beforeCrossChainMessageForwarding` in LIP 0051 to check `messageFeeTokenID`

### PR DESCRIPTION
**Pull request by mjerkov**
Resolves [Issue 425](https://github.com/LiskHQ/lips/issues/425)

### CODE REVIEW

File: proposals/lip-0051.md [hunk 1]
```diff
@@ -2540,6 +2540,8 @@ The following steps are executed as part of the execution of a cross-chain updat
 def verifyCrossChainMessage(trs: Transaction, ccm: CCM) -> None:
     # This is getTokenIDLSK() for channels with the mainchain.
     messageFeeTokenID = Interoperability.getMessageFeeTokenID(ccm.sendingChainID)
+    if messageFeeTokenID != 0x0000000000000000:
```
Comment (ricott1) [inline proposals/lip-0051.md:2543]:
```suggestion
    if messageFeeTokenID != getTokenIDLSK():
```

File: proposals/lip-0051.md [hunk 2]
```diff
@@ -2540,6 +2540,8 @@ The following steps are executed as part of the execution of a cross-chain updat
 def verifyCrossChainMessage(trs: Transaction, ccm: CCM) -> None:
     # This is getTokenIDLSK() for channels with the mainchain.
     messageFeeTokenID = Interoperability.getMessageFeeTokenID(ccm.sendingChainID)
+    if messageFeeTokenID != getTokenIDLSK():
```
Comment (ricott1) [inline proposals/lip-0051.md:2543]: This check should be cast only on mainchain, as on a sidechain the message fee can be anything (it must always be a native token)

**Review by sergeyshemyakov — APPROVED** [review]
Looks good to me

File: proposals/lip-0051.md [hunk 3]
```diff
@@ -2585,6 +2585,8 @@ def beforeCrossChainMessageForwarding(trs: Transaction, ccm: CCM, ccmFailed: boo
     if ccm.fee > 0:
         # This should never fail since it is checked in verifyCrossChainMessage.
         messageFeeTokenID = Interoperability.getMessageFeeTokenID(ccm.receivingChainID) # Always equal to getTokenIDLSK().
+        if messageFeeTokenID != getTokenIDLSK():
```
Comment (ricott1) [inline proposals/lip-0051.md:2588]: I would remove the comment `# Always equal to getTokenIDLSK().` from the line above

**Review by ricott1 — APPROVED** [review]

File: proposals/lip-0051.md (hunk 3 above)
Comment (mjerkov) [reply proposals/lip-0051.md:2588]: Done.

**Review by janhack — APPROVED** [review]
Editorial approval

**Pull request merged**